src/lib/moment/momentUtils.ts

```diff
@@ -6,6 +6,18 @@ export const formatUTCDateStringToLocal = (dateStringUTC: string): string => {
   return moment(dateStringUTC).local().format('YYYY-MM-DD HH:mm:ss');
 };
 
+
+export const lessThanCurrentPeriod = (dateString: string): boolean => {
+  const currentDate = moment(dateString);
+  const currentMonth = moment().startOf('month');
+  const currentDatePeriod = currentDate.valueOf();
+  const currentPeriod = currentMonth.valueOf();
+  if (currentDatePeriod < currentPeriod) {
+    return true;
+  }
+  return false;
```

Comment:
instead of `if (...) return true else false` just return the evaluation of the if statement